tests/pytest_tests/system_tests/test_core/test_public_api.py

```diff
@@ -736,3 +736,31 @@ def test_query_user_multiple(relay_server, inject_users):
         api = Api()
         assert api.user(email).email == email
         assert len(api.users(email)) == 2
+
+
+def test_create_run_queue_template_variables_not_supported(runner, user, monkeypatch):
```

Comment:
what is your thinking about moving launch tests to the launch dir?

Reply:
sure, I can put this test in a launch dir, but I was thinking this is a test of the public api.

Reply:
yeah it is... just wondering if it is more helpful to have the public queries that relate strictly to launch be in one place for easier discovery. 
i guess on the flip side it has the same file name... 
so really up to you